packages/config/config/devices/0x015f/mh-s212.json

```diff
@@ -21,5 +25,188 @@
 			"maxNodes": 5,
 			"isLifeline": true
 		}
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Restore State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"2": {
+			"label": "External Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Momentary Buttons",
+					"value": 0
+				},
+				{
+					"label": "2-State Switches",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "2-State Switches",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Toggle Switch without specified on/off position",
+					"value": 0
+				},
+				{
+					"label": "Toggle Switch with specified on/off position",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Switch 1 - Operation Report to Association Group 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Report both on/off",
+					"value": 0
+				},
+				{
+					"label": "Report on only",
+					"value": 1
+				},
+				{
+					"label": "Report off only",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Switch 1 - Off Timer",
```

Comment:
This label doesn't seem to match the options.

Reply:
have fixed